source/jormungandr/jormungandr/scenarios/simple.py

```diff
@@ -346,6 +346,13 @@ def _add_ridesharing_link(self, resp, params, instance):
         req['region'] = instance.name
         add_link(resp, rel='ridesharing_journeys', **req)
 
+    def _add_bypass_disruptions_link(self, resp, params):
+        # find first impact wit a NO_SERVICE severity
+        found = next((True for impact in resp.impacts if impact.severity.effect == Severity.NO_SERVICE), False)
+        if found:
+            params['data_freshness'] = 'realtime'
+            add_link(resp, rel='bypass_disruptions', **params)
```

Comment:
A suggestion to avoid mutating `params` :
```suggestion
            cloned_params = copy.deepcopy(params)
            cloned_params['data_freshness'] = 'realtime'
            add_link(resp, rel='bypass_disruptions', **cloned_params)
```

Reply:
:+1:

Reply:
Thanks !